### PR TITLE
Validate that catalog exists in CreateSchemaTask

### PR DESCRIPTION
**Pull request by silly-carbon**
## Description
Validate that catalog exists in CreateSchemaTask.

Implement TODO: validate that catalog exists

> Is this change a fix, improvement, new feature, refactoring, or other?
improvement
> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)
a change to the core query engine

## Documentation

( ) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

( ) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: core/trino-main/src/main/java/io/trino/execution/CreateSchemaTask.java [hunk 1]
```diff
@@ -88,7 +89,9 @@ static ListenableFuture<Void> internalExecute(
     {
         CatalogSchemaName schema = createCatalogSchemaName(session, statement, Optional.of(statement.getSchemaName()));
 
-        // TODO: validate that catalog exists
+        if (!plannerContext.getMetadata().catalogExists(session, schema.getCatalogName())) {
```
Comment (lukasz-stec) [inline core/trino-main/src/main/java/io/trino/execution/CreateSchemaTask.java:92]: Could you add a test case in `TestCreateSchemaTask`?

**Review by lukasz-stec — APPROVED** [review]
lgtm % test

**Comment by silly-carbon** [comment]
@lukasz-stec Thanks, I've added a test case, please take a look

**Review by lukasz-stec — APPROVED** [review]
lgtm

**Comment by lukasz-stec** [comment]
@findepi this is ready for a maintainer to review.

**Comment by findepi** [comment]
@silly-carbon  can you please make sure you filled the CLA? For more information, see https://github.com/trinodb/cla.

**Comment by silly-carbon** [comment]
Yes,  I have sent the PDF days ago. PTAL at the CLA mailbox😂



发自我的小米
在 Piotr Findeisen ***@***.***>，2022年8月26日 18:51写道：

@silly-carbon<https://github.com/silly-carbon> can you please make sure you filled the CLA? For more information, see https://github.com/trinodb/cla.

**Comment by findepi** [comment]
> Yes, I have sent the PDF days ago. PTAL at the CLA mailbox😂

@martint please take a look

**Comment by silly-carbon** [comment]
@martint Hello there, please take a look at the CLA mailbox 😂

**Comment by ebyhr** [comment]
@cla-bot check

**Comment by silly-carbon** [comment]
@findepi 
I find that there are 2 cancelled CI checks, could you please re-run those 2 checks? I believe that those ones would pass through this time (I couldn't find a place to re-run checks by myself)

**Comment by ebyhr** [comment]
Superseded by #12990

**Pull request closed without merging**